modules/account-lib/src/coin/near/transaction.ts

```diff
@@ -184,6 +240,25 @@ export class Transaction extends BaseTransaction {
     };
   }
 
+  /**
+   * Returns a complete explanation for a staking activate transaction
+   * @param {TxData} json The transaction data in json format
+   * @param {TransactionExplanation} explanationResult The transaction explanation to be completed
+   * @returns {TransactionExplanation}
+   */
+  explainStakingActivateTransaction(json: TxData, explanationResult: TransactionExplanation): TransactionExplanation {
```

Comment:
I don't know if for this case we only use explain tx and do each explain using the tx type. We need to think about this in order to have splitted the explaintx method.  Check other coins.